backend/src/Entity/User.php

```diff
@@ -281,4 +282,51 @@ public function updateWeights($postArray): self {
     
     return $this;
     }
+
+     /**
+     * Returns the roles granted to the user.
+     *
+     * <code>
+     * public function getRoles()
+     * {
+     * return array('ROLE_USER');
+     * }
+     * </code>
+     *
+     * Alternatively, the roles might be stored on a ``roles`` property,
+     * and populated in any number of different ways when the user object
+     * is created.
+     *
+     * @return (Role|string)[] The user roles
+     */
+    public function getRoles()
+    {
+    return array('ROLE_USER');
+    }
+
+    /**
+     * Returns the salt that was originally used to encode the password.
+     *
+     * This can return null if the password was not encoded using a salt.
+     *
+     * @return string|null The salt
+     */
+    public function getSalt()
+    {
+    return null;
+    }
+
+    /**
+     * Removes sensitive data from the user.
+     *
+     * This is important if, at any given point, sensitive information like
+     * the plain-text password is stored on this object.
+     */
+    public function eraseCredentials()
+    {
+        
+    }  
+
+    public function getUsername() {}
+
```

Comment:
PhpStan won't like those empty functions. If you have the solution faster than me, let me know 😉

Reply:
I know, already encountered the problem. Tried to install phpstan extensions only to get even more error messages....